docs/synapse/userguides/storm_ref_automation.ipynb

```diff
@@ -11,30 +43,675 @@
     "Storm Reference - Automation\n",
     "============================\n",
     "\n",
+    ".. _auto-bkgd:\n",
+    "\n",
+    "Background\n",
+    "----------\n",
+    "\n",
+    "Synapse is designed to facilitate large-scale analysis over disparate data sources with speed and efficiency. Many of the features that support this analysis are built into Synapse’s architecture, from performance-optimized indexing and storage to an extensible data model that allows you to reason over data in a structured manner.\n",
+    "\n",
+    "Synapse also supports large-scale analysis through the use of **automation.** Synapse’s automation features are available through the Storm runtime and include:\n",
+    "\n",
+    "- `Triggers and Cron`_\n",
+    "- `Macros`_\n",
+    "\n",
+    "Triggers, cron, and macros in Synapse all utilize the Storm query language. That is, regardless of the type of automation used, the result is the execution of a predefined Storm query. This means that **anything that can be written in Storm can be automated,** from the simple and straightforward to the highly complex. Actions performed via automation are limited only by imagination and Storm proficiency. Some automation is fairly simple (\"if X occurs, do Y\" or \"once a week, update Z\"). However, automation can take advantage of all available Storm features, including subqueries, variables, libraries, control flow logic, and so on.\n",
+    "\n",
+    ".. _auto-consid:\n",
+    "\n",
+    "Considerations\n",
+    "--------------\n",
+    "\n",
+    "This section is not meant to be a detailed guide on implementing automation. However, a few areas are included here for consideration when utilizing automation in your environment.\n",
+    "\n",
+    "**Permissions.**\n",
+    "\n",
+    "The ability to create, modify, or delete triggers, cron jobs, and macros depends on a user having the appropriate permissions to do so. If you assign permissions for these tasks in your environment, keep the following in mind:\n",
+    "\n",
+    "- Triggers and cron jobs execute in the context of (with the permissions of) **the user who creates them.**\n",
+    "\n",
+    "- Macros execute in the context of **the user who calls them** for execution, regardless of who created them.\n",
```

Comment:
Dmons execute in the context of the user that created it, in the view it was created in.